src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseDataParent.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.probate.model.ccd.raw.request;
+
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+@Jacksonized
+@SuperBuilder
+@Data
+public class CaseDataParent {
+
+    protected String registrySequenceNumber;
+
+}
```

Comment:
COULD: could have a tri class hieararchy, rename parent to CaseDataBase and fill that with all existing fields. Then  add a class that extends that, possibly called AdditionalCaseDataFields which would include new field, then have a class that extends that called CaseData with no fields